dll/win32/fltlib/fltlib.c

```diff
@@ -21,17 +21,24 @@
 
 /* DATA ****************************************************************************/
 
+#define NT_FACILITY(Status) ((Status >> 16) & 0xFFF)
```

Comment:
This looks a lot like `SCODE_FACILITY` (but not entirely, that uses `0x1FFF`) is that intentional?
Can't you use `SCODE_FACILITY` here?

Reply:
`SCODE_FACILITY` should not be used here, because:
- Status is not HResult, `SCODE_FACILITY` should be use to resolve HResult only.
- Facility code takes 12bits, a higher bit have special meaning, should not be ignored. See: https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-erref/87fba13e-bf06-450e-83b1-9241dc81e781

Windows SDK have no macro to get facility code of status, so we need to do something like Process Hacker here:
https://processhacker.sourceforge.io/doc/ntbasic_8h.html#ab922cd64630fe14a55627b1d987f644a

Reply:
Note that the "SCODE" stuff is a status code used in MAPI interface only.

Reply:
You are right, I confused `SCODE` and`HRESULT`, previous my comment fixed